packages/muffincss-core/src/processors/at-rules.ts

```
@@ -1,39 +1,25 @@
   constructor(
     resultCollector: ResolvedClassListCollector,
     options: Options["options"],
   ) {
     super(resultCollector, options);
-    this.parsedAtRules = new Map<string, Map<string, AtomicRule>>();
-  }
-  initParsedAtRules(atRuleParams: string) {
-    if (!this.parsedAtRules.has(atRuleParams)) {
-      this.parsedAtRules.set(atRuleParams, new Map<string, AtomicRule>());
-    }
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)